doc/reference/std/getopt.md

```diff
@@ -1,188 +1,17 @@
 # Command Line Argument Parsing
 
-The `:std/getopt` library provides facilities for command line argument parsing.
+This is the old name of the `:std/cli/getopt` module,
+that provides facilities for command line argument parsing.
 
-::: tip usage
-(import :std/getopt)
-:::
-
-## Interface
-
-### getopt
-```scheme
-(getopt <specifier> ...)
-=> <parser>
-
-specifier:
- (command id [help: text] <cmd-specifier>)
- <cmd-specifier> ...
-
-cmd-specifier:
- (flag id short [long])
- (option id short [long] [help: text] [value: proc] [default: value])
- (argument id [help: text] [value: proc])
- (optional-argument id [help: text] [value: proc] [default: value])
- (rest-arguments id [help: text] [value: proc])
-
-```
-
-`getopt` creates a command line parser, which can be used to parse arguments
-with `getopt-parse`.
-
-### getopt-parse
-```scheme
-(getopt-parse <parser> args)
-=> (values cmd-id options)
-   options
-```
-
-`getopt-parse` accepts a parser and a list of string arguments and parses
-according to the parser specification. If it is parsing a specification with
-subcommands, it returns two values, the command id and a hash table with the
-parsed options. Otherwise it just returns the hash table with the parsed options.
-An exception is raised if parsing the arguments fails.
-
-### getopt-error?
-```scheme
-(getopt-error? obj)
-=> boolean
-```
-
-If parsing fails, then a `getopt-error` is raised, which can be guarded with
-`getopt-error?`.
-
-### getopt-display-help
-```scheme
-(getopt-display-help <tip> program-name [port = (current-output-port)])
-
-
-tip:
- <getopt-error>
- <parser>
- <command>
-```
-
-The procedure `getopt-display-help` can be used to display
-a help message for a getopt error according to the argument specification.
+As of Gerbil v0.19 this name is deprecated and
```

Comment:
ah it is here. It is not really deprecated, just moved.
And the old library is the same as the new library, so this comment is misleading.